cmd/main.go

```diff
@@ -1,34 +1,41 @@
 package main
 
 import (
-	"flag"
 	"fmt"
 	"os"
+	"strings"
 	"weather"
 )
 
-type Response struct {
-	OneWord string  `json:"oneword"`
-	Celcius float64 `json:"celcius"`
+func LocationFromArgs(args []string) string {
+
+	var command strings.Builder
```

Comment:
Would it be easier to use `strings.Join` to turn `args` into a space-separated string?